eodag/resources/providers.yml

```diff
@@ -2782,6 +2784,16 @@
               "hydrological_year": {completionTimeFromAscendingNode#get_hydrological_year}
             }}
           - '{$.completionTimeFromAscendingNode#get_hydrological_year}'
+    FIRE_HISTORICAL:
```

Comment:
Why did you re-add FIRE_HISTORICAL to cop_cds? It has been removed from there (and moved to cop_ewds).